app/assets/javascripts/Components/submission_file_manager.jsx

```diff
@@ -254,9 +258,67 @@ class SubmissionFileManager extends React.Component {
     );
   };
 
+  renderRequiredFiles = () => {
+    let requiredFilesBox;
+
+    if (this.state.requiredFiles.length > 0) {
+      requiredFilesBox = (
+        <div>
+          <h2>{I18n.t("activerecord.attributes.assignment.assignment_files")}</h2>
+          <p>
+            {this.state.numberOfMissingFiles === 0 ? (
+              <p>{I18n.t("student.submission.all_files_submitted")}</p>
```

Comment:
You don't need the `p` tag here, since there's a `p` tag wrapping this entire part (two lines up).